assets/tests/components/appStateWrapper.test.tsx

```diff
@@ -13,7 +14,15 @@ jest.mock("userTestGroups", () => ({
   default: jest.fn(() => []),
 }))
 
-test("renders", () => {
+jest.mock("../../src/api")
+
+beforeEach(() => {
+  jest.mocked(fetchRoutes).mockImplementation(() => new Promise(() => {}))
+  jest.mocked(putRouteTabs).mockImplementation(() => new Promise(() => {}))
+  jest.mocked(fetchDetour).mockImplementation(() => new Promise(() => {}))
```

Comment:
I think you wanna use `neverPromise` here, and `mockReturnValue`

Reply:
thanks!